controller/src/controller/config.py

```diff
@@ -527,6 +582,120 @@ class PrepullerConfig(PrepullerOptions):
     source: DockerSourceConfig | GARSourceConfig
 
 
+class IndividualImageClassPolicy(BaseModel):
+    """Policy for images to display within a given class.
+
+    The policy has a 'number', an 'age', and a 'cutoff_version' field.  All
+    are optional.
```

Comment:
This is obvious from the code and from any documentation Sphinx will generate, so you don't have to write it in the docstring.